Clarify discrete aspect in README (#117)
gdalle/HiddenMarkovModels.jl: README.md

[![DOI](https://joss.theoj.org/papers/10.21105/joss.06436/status.svg)](https://doi.org/10.21105/joss.06436)

> A Julia package for simulation, inference and learning of Hidden Markov Models for which the hidden state has discrete rather than continuous values.

Reviewer: ```suggestion
A Julia package for simulation, inference and learning of Hidden Markov Models with discrete states and discrete time.
```
